### DISCUSSION TITLE
Identifying Network Names & Types

### DISCUSSION THREAD

**Discussion by santaviccap** [CMSgov/price-transparency-guide#619] [Q&A]
Hello All - 

I'm trying to understand if there is a standard approach users should be taking to understand what type of Network a particular INN file is representing.  The Plan name, as defined in the Schema is: "The plan name and name of plan sponsor and/or insurance company" which is unfortunately ambiguous, but probably pragmatic in certain ways also. What we're seeing in the files is that some Payers put the name of the ASO employer group in that field, some put what appears to be the name of the health insurance product (which may describe the benefits layered on top of the underlying network, not the network itself), and some put the Payer name.  None of these reliably describe the actual Network which is to be represented in the file.  

I've noticed that for some INN files, the URL itself appears to contain a name that is helpful in deciphering which type of Network that the file is describing.  For instance, some INN URL's clearly contain 'PPO', 'HMO', 'EPO' and so on so a consumer of the data could deduce that the file represents a PPO Network for the Payer in question.  Most do not. 

It appears that there really isn't a good spot in the Schema itself that requires Payers to, in a consistent and understandable format, indicate what Network or Network type they are describing in the file. 

Is this something that CMS is considering a change or update to in the future?

Not to pick on a particular Payer, but to take Anthem for an example - they put the employer group name in the "Plan name" field.  The URL's to their actual INN files contain what appears to be an internal identifier that describes the network in the file.  In the example below, it looks like "58B0" is what is describing this Network (890 is the BCBS plan code for TN).  Assuming this is an internal identifier, there is effectively no way to determine which Network this file is describing.  This seems to be a pretty big gap in the schema's ability to meet the intent of the regulation. 

"https://anthembcca.mrf.bcbs.com/2022-07_890_58B0_in-network-rates

One more question to add - is there a way to identify whether a Network is commercial or MA/Managed Medicaid?

Very much appreciate any insights here.  @shaselton-usds

**Comment by Rafael-Mora** [comment]
I have the same question, and I am particularly interested in understanding how we can identify between Commercial, Medicaid and Medicare. Any approach that has worked for you?

**Comment by Ginger228** [comment]
As a reminder Medicaid and Medicare is out of scope for Transparency in Coverage.
